petastorm/spark/spark_dataset_converter.py

```diff
@@ -454,6 +454,22 @@ def _check_url(dir_url):
             'Please prepend "file://" for local filesystem.'.format(dir_url))
 
 
+def _normalize_databricks_dbfs_url(url, err_msg):
```

Comment:
Can you please add a unittest for this function. Should be pretty straight forward.